src/index.html

```diff
@@ -9,6 +9,65 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="header__a">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li>
+            <a href="#apple" class="is-active nav__a"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML

Reply:
is not fixed